Formula/k3d.rb

```diff
@@ -18,18 +18,13 @@ class K3d < Formula
   def install
     system "go", "build",
            "-mod", "vendor",
-           "-ldflags", "-s -w -X github.com/rancher/k3d/version.Version=v#{version}",
+           "-ldflags", "-s -w -X github.com/rancher/k3d/v3/version.Version=v#{version}"\
+           " -X github.com/rancher/k3d/v3/version.K3sVersion=latest",
            "-trimpath", "-o", bin/"k3d"
     prefix.install_metafiles
   end
 
   test do
-    assert_match "k3d version v#{version}", shell_output("#{bin}/k3d --version")
-    code = if File.exist?("/var/run/docker.sock")
-      0
-    else
-      1
-    end
-    assert_match "Checking docker...", shell_output("#{bin}/k3d check-tools 2>&1", code)
```

Comment:
With the Docker check removed, it would be great if we could restore some basic coverage and add another test beyond a mere version check.
Can you think of one?

Reply:
Phew... All the other commands are actually executing something. We could add an enforced error check if that's ok?🤔

Reply:
Sounds good 👍

Reply:
Added a second test which (should) always fails